### PR TITLE
ScreenshotViewer action: open web, edit, share

### PR DESCRIPTION
**Pull request by Daisy-pliu**
1. [TC0054] open web of a screenshot 
2. [TC0055] edit a screenshot 
3. [TC0056] share a screenshot
4. [TC0057] view info a screenshot

New function
1. Before: Insert screenshot to database
2. After: delete screenshot from database

### CODE REVIEW

**Comment by mTwTm** [comment]
Please rebase to current firefoxlite/master to get rid of a test intermittent.

**Comment by Daisy-pliu** [comment]
rebased

File: app/src/androidTest/java/org/mozilla/focus/autobot/Autobot.kt [hunk 1]
```diff
@@ -74,6 +80,51 @@ class ScreenshotRobot : MenuRobot() {
...
+                .check(ViewAssertions.doesNotExist())
+    }
+
+    fun clickOpenWebInScreenshotViewer(url: String) {
+
+        // Click open web
+        onView(withId(R.id.screenshot_viewer_btn_open_url)).perform(click())
+
+        //Check Url displayed
+        onView(AllOf.allOf(withId(R.id.display_url), isDisplayed())).check(matches(withText(url)))
```
Comment (cnevinc) [inline app/src/androidTest/java/org/mozilla/focus/autobot/Autobot.kt]: AllOf.allOf

**Comment by Daisy-pliu** [comment]
runs over time constrain

**Pull request closed without merging**